File: Psets/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
  int n = 0;

    do
    {
        n = get_int("height: ");
    }
    while (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        for (int j = n - i; j > 1; j--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

    return 0;
}

File: Psets/pset6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

# ciclo infinito
while True:
    # leer altura de la piramide
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
# iterar sobre la altura
for row in range(height):
    # impresion de los espacios
    for space in range(height - row - 1, 0, -1):
        print(" ", end="")
    # impresion del hash
    for hash in range(row + 1):
        print("#", end="")
    # salto de linea
    print("\n", end="")